useManageLabels.test.tsxでfetchLabelsモック未設定により警告が出る

## 背景\n ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/private/var/folders/sq/rxs6kphj17g1wwx8wz42v06r0000gn/T/vibe-kanban/worktrees/cbc6-frontend-src-fea". 実行時、  でモックした  が戻り値を返さず  が  を受け取り、React Query が  を出力しています。\n\n## 原因\n- テストで  の戻り値を設定していないため  が  を返す。\n\n## 対応方針\n- テストのセットアップで  を  もしくはモックデータで返す。\n- あるいは  などで初期値を注入する。\n\n## 再現手順\n1. \n2.  ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/private/var/folders/sq/rxs6kphj17g1wwx8wz42v06r0000gn/T/vibe-kanban/worktrees/cbc6-frontend-src-fea".\n\n## 期待結果\n- テスト実行時に警告が出ない状態。

--- a/frontend/src/features/labels/hooks/useManageLabels.test.tsx
+++ b/frontend/src/features/labels/hooks/useManageLabels.test.tsx
@@ -5,13 +5,13 @@ import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { act, renderHook } from "@testing-library/react";
 import type { ReactNode } from "react";
 import { beforeEach, describe, expect, test, vi } from "vitest";
-import * as api from "../queries/api";
 import type { Label } from "../types";
 import { useManageLabels } from "./useManageLabels";
 
 // API関数のモック
 vi.mock("../queries/api", () => ({
-	fetchLabels: vi.fn(),
+	// React Queryの警告を防ぐため、デフォルトの戻り値を設定
+	fetchLabels: vi.fn().mockResolvedValue([]),
 	createLabel: vi.fn(),
 	updateLabelDescription: vi.fn(),
 	deleteLabel: vi.fn(),
@@ -35,8 +35,6 @@ describe("useManageLabels", () => {
 			},
 		});
 		vi.clearAllMocks();
-		// fetchLabelsモックのデフォルト戻り値を設定（警告を防ぐため）
-		vi.mocked(api.fetchLabels).mockResolvedValue([]);
 	});
 
 	function createWrapper() {

